packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -549,7 +549,8 @@ class AndroidDevice extends Device {
       observatoryDiscovery = ProtocolDiscovery.observatory(
         getLogReader(),
         portForwarder: portForwarder,
-        hostPort: debuggingOptions.observatoryPort,
+        hostPort: debuggingOptions.hostVmservicePort,
```

Comment:
How do we format Vmservice in other places? Should we do VMService, VmService, or keep Vmservice?

Reply:
Updated to consistently use `VmService`